### PR TITLE
11876 implement overall flow

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Just wiring up some stuff 

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11876

### CODE REVIEW

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Review by pascal-brause — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/ChangeAdmissionScenario/ChangeAdmissionScenarionCellModel.swift [hunk 1]
```diff
@@ -7,10 +7,17 @@ import OpenCombine
 
 class ChangeAdmissionScenarionCellModel: AddButtonAsTableViewCelling {
 	
+	// MARK: - Init
+
+	init(
+		changeAdmissionScenarioButtonLabel: String
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/ChangeAdmissionScenario/ChangeAdmissionScenarionCellModel.swift:13]:
```suggestion
		changeAdmissionScenarioButtonText: String
```

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/HealthCertificateOverviewViewModel.swift [hunk 2]
```diff
@@ -53,6 +66,8 @@ class HealthCertificateOverviewViewModel {
 	@DidSetPublished var decodingFailedHealthCertificates: [DecodingFailedHealthCertificate] = []
 	@DidSetPublished var testCertificateRequests: [TestCertificateRequest] = []
 	@DidSetPublished var testCertificateRequestError: HealthCertificateServiceError.TestCertificateRequestError?
+	@DidSetPublished var changeAdmissionScenarioStatusLabel: DCCUIText?
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/HealthCertificateOverviewViewModel.swift:69]:
```suggestion
	@DidSetPublished var changeAdmissionScenarioStatusText: DCCUIText?
```

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/HealthCertificateOverviewViewModel.swift [hunk 3]
```diff
@@ -53,6 +66,8 @@ class HealthCertificateOverviewViewModel {
...
+	@DidSetPublished var changeAdmissionScenarioButtonLabel: DCCUIText?
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/HealthCertificateOverviewViewModel.swift:70]:
```suggestion
	@DidSetPublished var changeAdmissionScenarioButtonText: DCCUIText?
```

**Review by ArturFriesen — APPROVED** [review]

**Pull request merged**